integration-cli/docker_utils.go

```diff
@@ -679,6 +679,22 @@ func dockerCmdInDirWithTimeout(timeout time.Duration, path string, args ...strin
 	return out, status, err
 }
 
+func findContainerExitCode(c *check.C, name string, vargs ...string) (string, int) {
+	args := append(vargs, "inspect", "--format='{{ .State.ExitCode }} {{ .State.Error }}'", name)
+	cmd := exec.Command(dockerBinary, args...)
+	out, _, err := runCommandWithOutput(cmd)
+	if err != nil {
+		c.Fatal(err, out)
+	}
+	// return 127 for Windows build testing
```

Comment:
No idea.... But FWIW, on master I see that Windows is perhaps already doing what you're wanting. docker run busybox foo, pointing to a Linux box. State.ExitCode=-1 on both platforms. echo %ERRORLEVEL% on Windows gives 127. But yes, echo $? on Linux is 1. 

Repeating same on master but using your /etc example. From a both CLI, State.ExitCode=-1. Linux $?=1, Windows %ERRORLEVEL%=1.